src/main/java/cloud/eppo/ShardUtils.java

```diff
@@ -1,18 +0,0 @@
-package cloud.eppo;
-
-import cloud.eppo.model.ShardRange;
-
-public final class ShardUtils {
-  /** Returns whether the given shard is within the range. */
-  public static boolean isShardInRange(int shard, ShardRange range) {
-    return shard >= range.getStart() && shard < range.getEnd();
-  }
-
-  public static int getShard(String input, int maxShardValue) {
```

Comment:
THis method was both rewritten and moved to `Utils` to cluster it with the similar `getMD5Hex()`